data/domain/stamps.tsx

```diff
@@ -309,6 +313,7 @@ export function updateStamps(data: Map<string, any>) {
         })
     }
 
+    const limitIsRemoved = sneaking.jadeUpgrades.find(upgrade => upgrade.index == 0)?.purchased ?? false;
```

Comment:
I think the correct index is 5, unless I'm missing something 🤔 
```suggestion
    const limitIsRemoved = sneaking.jadeUpgrades.find(upgrade => upgrade.index == 5)?.purchased ?? false;
```